orangecontrib/text/widgets/owwordcloud.py

```diff
@@ -215,9 +217,23 @@ def _apply_topic(self):
         self._repopulate_wordcloud(words, weights)
 
     def _apply_corpus(self):
+        words, freq = self.word_frequencies()
+        self._repopulate_wordcloud(words, freq)
+
+    def word_frequencies(self):
         counts = self.corpus_counter.most_common()
         words, freq = zip(*counts) if counts else ([], [])
-        self._repopulate_wordcloud(words, freq)
+        return words, freq
+
+    def create_weight_list(self):
+        words, freq = self.word_frequencies()
+        words = np.array(words)[:, None]
+        weights = np.array(freq)[:, None]
+        domain = Domain([ContinuousVariable('Weight')],
+                        metas=[StringVariable('Word')])
+        weight_table = Table.from_numpy(domain, X=weights, metas=words)
+        weight_table.name = 'weights'
```

Comment:
Same comment as above. Also I would camel case this, if it is not agains some rule we have.